package.json

```diff
@@ -25,7 +25,6 @@
     "@vue/component-compiler": "^4.2.4",
     "@vueuse/core": "^9.5.0",
     "ace-builds": "^1.4.8",
-    "air-datepicker": "github:frappe/air-datepicker",
```

Comment:
We agreed that we still need this dependency for the DateRange control, right?

Reply:
Yes, if we want to keep this form. As mentioned, it is only used in one report. And IMHO it would be better to remove it completely, but this up to the dev-team.